src/main/java/de/tum/in/www1/artemis/web/rest/dto/ParticipantScoreAverageDTO.java

```diff
@@ -0,0 +1,30 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import de.tum.in.www1.artemis.domain.Team;
+import de.tum.in.www1.artemis.domain.User;
+
+public class ParticipantScoreAverageDTO {
+
+    public String userName;
+
+    public String teamName;
+
+    public Double averageScore;
+
+    public Double averageRatedScore;
+
+    public ParticipantScoreAverageDTO(User user, Double averageScore, Double averageRatedScore) {
+        this.userName = user.getLogin();
+        this.teamName = null;
+        this.averageScore = averageScore;
+        this.averageRatedScore = averageRatedScore;
+    }
+
+    public ParticipantScoreAverageDTO(Team team, Double averageScore, Double averageRatedScore) {
+        this.userName = null;
```

Comment:
you are using null instead of undefined. Is this intended?

Reply:
Yes. In this case here yes. I wrote the server code such that all the properties are sent always. The object is used in the table as is, just dropping some properties makes it harder to just display everything in a table. (i would have to always handle the case that every single property could potentially be undefined and so on)